### ISSUE TITLE
plot:  freerun stops at the end of an acquisition

### ISSUE THREAD

**Issue by ddkohler** [wright-group/yaqc-cmds#347]
Desired behavior is to resume freerun if the box is checked

**Comment by ddkohler** [comment]
sensor measure sets loop=False
https://github.com/wright-group/yaqc-cmds/blob/231151940caa8479956389244b66eb0a62faacd3/yaqc_cmds/sensors/_sensors.py#L165

acquisition calls measure
https://github.com/wright-group/yaqc-cmds/blob/231151940caa8479956389244b66eb0a62faacd3/yaqc_cmds/somatic/acquisition.py#L280

fix is likely just look up freerun state at acquisition finish, i.e.
```
            for s in yaqc_cmds.sensors.sensors:
                s.loop()
```

**Comment by ksunden** [comment]
closed by #357
